### PR TITLE
🔥Feat: Add a mew flow to code review

### PR DESCRIPTION
**Pull request by renanbastos93**
According to talk in another moment and an initial discussion on this issue https://github.com/gofiber/fiber/issues/289, I am proposing a new flow to Fiber accept pull requests in all repositories.

### CODE REVIEW

**Comment by thomasvvugt** [comment]
Hi @renanbastos93 ,
Thanks for your pull request on improving our Pull Request Templates.
However, during a review of your code I found some mismatches using the .github folder such as a typo in CONTRIBUTING.md.

Also, I think the core maintainers of Fiber; especially @Fenny needs to take a critical look at this PR.
I also do not think that having emoji's in each commit is preferred.

Nonetheless, thanks for your commitment!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!--
+Please, don't forget to use the prefixes in your title, following document https://github.com/gofiber/fiber/blob/master/.github/contribuiting.md.
```
Comment (jozsefsallai) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: Please, don't forget to use prefixes in your title, as described in the following document https://github.com/gofiber/fiber/blob/master/.github/contribuiting.md.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+
+e.g
+🔥Feat: Add flow to add person (title)
+
+Added to flow to add a person. (description your pull request)
```
Comment (jozsefsallai) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: We need a better example for this, something like:

```
📚 docs: Improve Middleware docs

Add an example on how to use the embed middleware
```

Also I prefer the old structure of the PR template

File: .github/contribuiting.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+- 🩹Fix: Improve flow
+- 🚨Test: Validate to add a new person
+- 📚Doc: Translate to Portuguese middleware redirect
+- 🎨Style: Respected pattern Golint
+
+All pull request that are a feature or fix is mandatory to have unit tests on the pull request.
+
+
+## Note
+Your pull request only to merge if you respecting this flow, even so, is required minimal 2 approval from members to Fiber.
```
Comment (jozsefsallai) [inline .github/contribuiting.md:36]: ## Prefixes to use in titles of your pull requests or commits
We will see now some examples of titles using the following mandatory prefixes:
> 🔥Feature, ♻️Refactor, 🩹Fix, 🚨Test, 📚Doc, 🎨Style

- 🔥Feat: Improve URL matcher
- ♻️Refactor: Rename file X to Y
- 🩹Fix: Improve flow
- 🚨Test: Write unit tests for the URL matcher
- 📚Doc: Translate the redirect middleware to Portuguese
- 🎨Style: Follow Golint pattern

All pull request that implement a feature or fix are required to have unit tests in the pull request.


## Note
Your pull request will only be merged if you respect this flow. In addition, it must get at least 2 approvals from members of the Fiber team.

**Comment by Fenny** [comment]
@renanbastos93 thank you for your input and great suggestions. This is something that was already in the process behind the scenes. I will take your ideas into consideration before I create a PR ❤️

**Pull request closed without merging**